textBasedAdventure.js

```diff
@@ -1,5 +1,190 @@
+
+
+const roomThree = () => {
+    console.log(" ")
+    userInput = readline.question("Congratulations, \
+    You and Ban enter Room 3" + " \
+    Room 3 contains six doors\
+     " + "make your pick " + "\
+     please enter a number between 1 and 6 ")
+        if(userInput === "1" ||userInput === "2"||userInput === "3") {
+            roomFour()
+        } else {    
+            console.log(fail)
+        }
+    }
+
+    const roomFour = () => {
+        console.log(" ")
+        userInput = readline.question("Props!, \
+        You and Ban enter Room 4" + " \
+        Room 4 contains eight doors\
+         " + "make your pick " + "\
+         please enter a number between 1 and 8 ")
+            if(userInput === "1" ||userInput === "3"||userInput === "5"||userInput === "6") {
+                roomFive()
+            } else {    
+                console.log(fail)
+            }
+        }
+
+
+        const roomFive = () => {
+            console.log(" ")
+            userInput = readline.question("Hip, hip, hurrah!, \
+            You and Ban enter Room 5" + " \
+            Room 5 contains ten doors\
+             " + "make your pick " + "\
+             please enter a number between 1 and 10 ")
+                if(userInput === "1" ||userInput === "3"||userInput === "8"||userInput === "9"||userInput === "5") {
+                    roomSix()
+                } else if (userInput === "7"){
+                    console.log(secret)
+                } else {    
+                    console.log(fail)
+                }
+            }
+
+
+
+            const roomSix = () => {
+                console.log(" ")
+                userInput = readline.question("Kudos!, \
+                You and Ban enter Room 6" + " \
+                Room 6 contains twelve doors\
+                 " + "make your pick " + "\
+                 please enter a number between 1 and 12 ")
+                    if(userInput === "5" ||userInput === "9") {
+                        roomSeven()
+                    } else {    
+                        console.log(fail)
+                    }
+                }
+
+
+                const roomSeven = () => {
+                    console.log(" ")
+                    userInput3 = readline.question("Hats off!, \
+                    You and Ban enter Room 7" + " \
+                    Room 7 contains a blue door, a yellow door, a red door, a green door, a black door, a white door, a brown door, a pink door, and a silver door\
+                     " + "make your pick " + "\
+                     please enter in one of the door colors ")
+                    //  for (let i = 0; i < arr.length; i+=3) 
+                     let i = 0
+                     while(i < arr.length){
+                        userInput3 === arr[i]
+                     }
+                     i+=3; {
+                        if(userInput3 = arr[i]) {
+                            break ;
```

Comment:
Had to remove 'break' to make it run. 

You are trying to break outside a while or for loop